ci-operator/step-registry/openshift/image-based/upgrade/target/gather/lca/openshift-image-based-upgrade-target-gather-lca-commands.sh

```diff
@@ -32,7 +32,7 @@ mkdir -p \$gather_dir_extra
 export KUBECONFIG=${target_kubeconfig}
 
 # Inspect the namespace
-oc adm inspect ns/openshift-lifecycle-agent --dest-dir=\$gather_dir
+oc adm must-gather --image=${LCA_PULL_REF} --dest-dir=\$gather_dir
 
 # Get installation service logs and recert summary files
```

Comment:
We are already collecting `installation-configuration.service` with lca must-gather...so i think it's safe to remove this